### ISSUE TITLE
Setup community.digitalocean

### ISSUE THREAD

**Issue by gundalow** [ansible-collections/community.digitalocean#1]
- [X] Create repo
- [X] GitHub Action (Just [sanity](https://github.com/ansible-collections/community.digitalocean/blob/main/.github/workflows/ansible-test.yml))
- [X] Add to Ansible package https://github.com/ansible-community/ansible-build-data/pull/14
- [x] README.md
- [x] galaxy.yml
- [x] Add to Zuul PR1 (gundalow)
- [x] Add to Zuul PR1 (gundalow)
- [x] Delete from community.general [PR1](https://github.com/ansible-collections/overview/blob/master/README.rst#q-what-should-i-do-to-move-plugins-across-collections-during-migration) ansible-collections/community.general#622
- [x] Add to this repo [PR2](https://github.com/ansible-collections/overview/blob/master/README.rst#q-what-should-i-do-to-move-plugins-across-collections-during-migration) 
- [X] ansible/ansible:devel [PR3](https://github.com/ansible-collections/overview/blob/master/README.rst#q-what-should-i-do-to-move-plugins-across-collections-during-migration) ansible/ansible#70483
- [x] Above PR backported to `ansible/ansible:stable-2.10` https://github.com/ansible/ansible/pull/70498

After the collection is live, we can add integration tests (via `ansible-core-ci`) with Shippable once we have credentials

**Comment by geerlingguy** [comment]
Just got my fork set up and was wondering, are there already integration tests for the DO modules? I have only tangentially contributed in the past but will help to try to get these out the door and maintained, at least (probably won't be contributing much outside of organization and help with releases).

**Comment by gundalow** [comment]
No integration tests yet, though it's on the list as Digital Ocean can provide us with some CI resources.

**Comment by felixfontein** [comment]
Basically @gundalow alrady said that, but the tests are all marked as `unsupported`, so they never ran in CI.

**Comment by geerlingguy** [comment]
Created https://github.com/ansible-collections/community.digitalocean/issues/15 as a follow-up for the tests.

I think for now, my goal is to get the docs examples for each module up to date (use FQCN, fix a few inconsistencies I've spotted), then release a 1.0.0 so we can have a stable branch to work from for 2.10.

Then we can start adopting new content, fixing other things, and deprecating the `_facts` modules in a 2.0.0 branch.

Closing this as the punchlist is complete.
